### ISSUE TITLE
Debian support

### ISSUE THREAD

**Issue by bluebob** [meltwater/puppet-cpan#28]
Trying to use module on Ubuntu 15.04 systems and getting the following message:

> Error: Evaluation Error: Error while evaluating a Function Call, Module cpan is not supported on Debian at /etc/puppetlabs/code/environments/production/modules/cpan/manifests/init.pp:59:7 on node kermit.

Tried this with the following small code fragment:

```
$cpanmods=['Net::SSLeay',]
include cpan
cpan { $cpanmods: 
  ensure => present, 
  require => Class['::cpan'],
}
```

Is there a problem or have I missed out something? Module was manually installed on this test server with the usual 'sudo puppet module install meltwater-cpan'.

**Comment by andrelaszlo** [comment]
I haven't used this module myself, but I have two follow-up questions for you:

1. Which version of puppet-cpan are you using? It seems like the line number of your error doesn't match the latest version. It seems to match [this](https://github.com/meltwater/puppet-cpan/blob/a17ae4a7dff16af67d4f474f8cd2b544a6714d5c/manifests/init.pp) version, but then the string `Debian` should have matched your OS family.
2. The module name doesn't seem to match your test code, is this the exact error message you got from this code snippet?

**Comment by andrelaszlo** [comment]
Was this fixed by #29 ? Ping @markuskont @alexgoodchild @bluebob

**Comment by markuskont** [comment]
Yes. Same problem.
